snuba/util.py

```diff
@@ -155,6 +155,23 @@ def is_function(column_expr: Any, depth: int=0) -> Optional[Tuple[Any, ...]]:
         return None
 
 
+def is_alias_column_expr(column_expr: Any) -> bool:
+    """
+    Returns True if column_expr is an alias column expression, otherwise False.
+
+    An alias column expression is a 3-tuple of (column_name, None, alias)
+    """
+    if (
+        isinstance(column_expr, (list, tuple))
+        and len(column_expr) == 3
+        and isinstance(column_expr[0], str)
+        and column_expr[1] is None
```

Comment:
Why do we need to keep this element in an aliased column ? it should be possible to distinguish an alaised column [col:str, alias:str] from a function [function, [parameters], alias:str]